packages/embeds/embed-core/src/constants.ts

```diff
@@ -1,6 +1,6 @@
 // We can't keep it too high because that might cause stale content to be shown
 // TODO: We should introduce a slotRefresh mechanism
-export const EMBED_MODAL_IFRAME_SLOT_STALE_TIME = 10 * 1000; // 1 minute
+export const EMBED_MODAL_IFRAME_SLOT_STALE_TIME = 60 * 1000; // 1 minute
```

Comment:
Increased it back to support broader usecases where actual CTA click could take some time to happen